quipucords/scanner/network/inspect_callback.py

```diff
@@ -191,6 +193,7 @@ def _finalize_host(self, host, host_status):
     @transaction.atomic
     def v2_runner_on_unreachable(self, result):
         """Print a json representation of the result."""
+        input_log.log_ansible_result(result)
```

Comment:
May want to try/catch around this so any IO or other related failure in this flow doesn't kill a scan.